.changeset/old-seahorses-switch.md

```diff
@@ -0,0 +1,5 @@
+---
+"@twilio-paste/website": patch
+---
+
+[Website]: Fix the flash of default theme on load and reload
```

Comment:
You don't need to do a changeset for any changes to the website. It is only for the core package that we publish to NPM